components/audit_table/audit_table.jsx

```diff
@@ -211,436 +211,448 @@ const holders = defineMessages({
     },
 });
 
-function AuditTable(props) {
-    var accessList = [];
-
-    const {formatMessage} = props.intl;
-    for (var i = 0; i < props.audits.length; i++) {
-        const audit = props.audits[i];
-        const auditInfo = formatAuditInfo(audit, formatMessage, props.currentUser, props.getByName, props.getUser);
-
-        let uContent;
-        if (props.showUserId) {
-            const profile = props.getUser(auditInfo.userId);
-            const data = profile ? profile.email : auditInfo.userId;
-            uContent = <td className='word-break--all'>{data}</td>;
-        }
+export class AuditTable extends React.PureComponent {
+    static propTypes = {
+        intl: intlShape.isRequired,
```

Comment:
Not super important, but probably we can use the `intl` from the context instead of expecting it from the props